### DISCUSSION TITLE
VcImageryProviderWmts属性設置問題

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#355] [Q&A] [answered]
您好,
VcImageryProviderWmts 使用本地的圖磚服務器(如下程式碼),會出現下方截圖的警告,
但属性都有設置,卻好像沒傳進去,是那裡設置有問題?

PS:另外使用new Cesium.WebMapTileServiceImageryProvider,以下方的設置是可以成功載入影像圖磚

![2023-02-16_163528](https://user-images.githubusercontent.com/5283322/219312223-d13dff1c-35a2-4d99-ba5e-392e0fc770d1.png)
![2023-02-16_162614](https://user-images.githubusercontent.com/5283322/219310024-42dc37bc-154c-4682-a6b1-fbb8ea497c7d.png)

**Comment by zouyaoji** [comment, marked as answer]
应该是你的开发环境不支持大写的属性。需要改成 wmts-style
tile-matrix-set-i-d

**Comment by stan88** [comment]
我用的是VS 2022 改小寫可以了,謝謝

**Reply by zouyaoji** [reply]
好的
